packages/frontend/src/components/pagination/pagination-ball/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { cva } from "class-variance-authority";
+import React, { MouseEventHandler, ReactNode } from "react";
+
+const paginationBallStyles = cva(
+    ["flex items-center justify-center", " w-12 h-12", "rounded-full"],
+    {
+        variants: {
+            isActive: {
+                true: "bg-green text-gray-800",
+                false: "bg-gray-200",
+            },
+            isClickable: {
+                true: "cursor-pointer hover:bg-green",
+            },
+        },
+    }
+);
+
+export const PaginationBall = ({
+    number,
+    onClick,
+    currentPage,
+    children,
+}: {
+    number?: number;
+    onClick?: MouseEventHandler<HTMLDivElement>;
+    currentPage?: number;
+    children?: ReactNode;
+}) => {
+    const isActive = Number(number) === currentPage;
+    const isClickable = !!onClick;
```

Comment:
I'd remove these two `const` and put the conditions directly in the `paginationBallStyles`.

Reply:
Yes, that might work better. thanks

Reply:
fixed: d404c18e7f424abd4115012d89d1e801d240a720